vm.go

```diff
@@ -225,6 +261,23 @@ func (vm *VM) EvaluateSnippetMulti(filename string, snippet string) (files map[s
 	return
 }
 
+// EvaluateSnippetDependencies evaluates a string containing Jsonnet code returning an array of dependencies.
+//
+// The filename parameter is only used for error messages.
+func (vm *VM) EvaluateSnippetDependencies(filename string, snippet string) (dependencies []string, formattedErr error) {
+	node, err := SnippetToAST(filename, snippet)
+	if err != nil {
+		return nil, err
+	}
+	var stackTrace []traceFrame
+	err = vm.evaluateDependencies(filename, parser.Children(node), &dependencies, &stackTrace, 0)
+	if err != nil {
+		err = makeRuntimeError(err.Error(), stackTrace)
```

Comment:
Not sure about error formatting here, especially about wrapping an error from vm.ResolveImport in a RuntimeError. How does the output look like if you run the command on a file with missing import (e.g. `import "foo.libsonnet"` when there is not `foo.libsonnet`)?

Reply:
```
$ cat bar.jsonnet
local foo = import 'foo.jsonnet';
foo

$ ~/go/bin/jsonnetdeps bar.jsonnet
RUNTIME ERROR: couldn't open import "foo.jsonnet": no match locally or in the Jsonnet library paths
        bar.jsonnet:1:13-33
```

Reply:
I was not sure whether missing imports should raise an error. One might want to simply ignore missing imports if they are wrapped in conditionals or such.